### PR TITLE
Add edit ico

### LINKED ISSUE #1622: Edit this page near the top.
From splitting #1212 

@AlexDaniel thinks it would improve things a lot to also have [an edit this page link] near the top of the page.

### PR DESCRIPTION
**Pull request by W4anD0eR96**
A try to fix #1622 .

### CODE REVIEW

**Comment by W4anD0eR96** [comment]
Is this satisfies issue #1622 ?

**Comment by AlexDaniel** [comment]
To your last comment: yes, if this is merged we can close #1622.

Building the docs currently to play with it. Right away I can tell that “Edit this page” text should be added somewhere. I probably don't mind it being alt text or something, but it has to be somewhere.

**Comment by AlexDaniel** [comment]
Looks a little bit differently here:
2017-11-09-113221_1329x431_scrot

**Comment by W4anD0eR96** [comment]
@AlexDaniel yes. do we have available js function to show message onmouseover?

I find GitHub pencil ico implements it by add `aria-label` attr inside `button`.
However, it's a Bootstrap feature.  I think we use jQuery and thus need to write a custom function.

**Comment by AlexDaniel** [comment]
Well, you can use ``title="Edit this page"`` and it will work, but I don't know what would be the best practice.

**Comment by W4anD0eR96** [comment]
@AlexDaniel  Is it?

I just check appearances of the ico locally in, all `/language/*`, all `/program/*`, all `/type/*` except most of `X::...`. Their looks are consistent. Could you check the html code relevant?

**Comment by AlexDaniel** [comment]
@W4anD0eR96 what I meant by that is that the button looks differently on your screenshot and mine.

**Comment by W4anD0eR96** [comment]
Yes, chrome and Safari look like mine, Firefox doesn't.

**Comment by W4anD0eR96** [comment]
Firefox:




Chrome and Safari:




@AlexDaniel Now the ico is quite familiar, and "Edit this page" message shows correctly.

However, I notice that our sass style render a litter different between browsers. If we will re-design the doc site, this should be taken into consideration.

I'll merge this PR once build passed.

File: html/images/pencil.svg [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="14px" height="16px" viewBox="0 0 14 16" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 40.3 (33839) - http://www.bohemiancoding.com/sketch -->
```
Comment (AlexDaniel) [inline html/images/pencil.svg]: I don't think we should pollute the repo with ads.

File: html/images/pencil.svg [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    <title>pencil</title>
+    <desc>Created with Sketch.</desc>
```
Comment (AlexDaniel) [inline html/images/pencil.svg]: Same here.

File: html/images/pencil.svg (hunk 1 above)
Comment (AlexDaniel) [reply html/images/pencil.svg]: (also, not only the repo, but every page we serve)

Comment (W4anD0eR96) [reply html/images/pencil.svg]: agree

**Pull request merged**